app/assets/javascripts/__tests__/search.test.js

```diff
@@ -0,0 +1,161 @@
+'use strict'
+
+// set the window.jQuery variable that the file expects
+window.jQuery = require('../../../../vendor/assets/javascripts/product-page-template/vendor/jquery.js')
```

Comment:
:( 

Yeah, I was going to have a look at doing this more robustly when fixing some of the asset compiling stuff. Although I _did_ think this was the jQuery we were using, so that's annoying.